src/components/List/List.module.css

```diff
@@ -0,0 +1,39 @@
+.list {
+  list-style-type: none;
+  padding: 0;
+  margin: 0;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  width: 100%;
+}
+
+.compact {
+  border-radius: 0.5rem;
+  background-color: #fff;
+}
+
+.spacious {
+  gap: 0.5rem;
+}
+.spacious > .listItem {
+  padding: 0.5rem;
+  border-radius: 0.5rem;
+}
+
+.listItem {
+  background-color: #fff;
+  list-style-type: none;
+  padding: 0;
+  margin: 0;
+  display: flex;
+  flex-direction: column;
```

Comment:
listItem currently has only one child, so flex-direction is not needed (perhaps not the other flex styling either)